.dockerignore

```diff
@@ -3,3 +3,5 @@
 build/_workspace
 build/_bin
 tests/testdata
+
+suave/devenv
```

Comment:
this seems wrong to ignore 🤔

Reply:
This is the dockerignore - we don't want to push the docker-compose.yml as that forces re-building of the geth docker image on every change to the devnet definition - huge time sink.

Reply:
lmao thought it was the gitignore. thxxx